public/javascripts/InitializeSubmittedLabels.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Initializes labels onto map/choropleth, returns information about label layers on map.
+ */
+function InitializeSubmittedLabels(map, params, adminGSVLabelView, mapData, labelData) {
+    var colorMapping = util.misc.getLabelColors();
+    var geojsonMarkerOptions = {
+            radius: 5,
+            fillColor: "#ff7800",
+            color: "#ffffff",
+            weight: 1,
+            opacity: 0.5,
+            fillOpacity: 0.5,
+            "stroke-width": 1
+        };
+
+    var auditedStreetColor = params.streetColor;
+    // Count a number of each label type
+    var labelCounter = {
+        "CurbRamp": 0,
+        "NoCurbRamp": 0,
+        "Obstacle": 0,
+        "SurfaceProblem": 0,
+        "NoSidewalk": 0
+    };
+    for (var i = labelData.features.length - 1; i >= 0; i--) {
+        labelCounter[labelData.features[i].properties.label_type] += 1;
+    }
+    document.getElementById("map-legend-curb-ramp").innerHTML = "<svg width='20' height='20'><circle r='6' cx='10' cy='10' fill='" + colorMapping['CurbRamp'].fillStyle + "'></svg>";
+    document.getElementById("map-legend-no-curb-ramp").innerHTML = "<svg width='20' height='20'><circle r='6' cx='10' cy='10' fill='" + colorMapping['NoCurbRamp'].fillStyle + "'></svg>";
+    document.getElementById("map-legend-obstacle").innerHTML = "<svg width='20' height='20'><circle r='6' cx='10' cy='10' fill='" + colorMapping['Obstacle'].fillStyle + "'></svg>";
+    document.getElementById("map-legend-surface-problem").innerHTML = "<svg width='20' height='20'><circle r='6' cx='10' cy='10' fill='" + colorMapping['SurfaceProblem'].fillStyle + "'></svg>";
+    document.getElementById("map-legend-no-sidewalk").innerHTML = "<svg width='20' height='20'><circle r='6' cx='10' cy='10' fill='" + colorMapping['NoSidewalk'].fillStyle + "' stroke='" + colorMapping['NoSidewalk'].strokeStyle + "'></svg>";
+    document.getElementById("map-legend-audited-street").innerHTML = "<svg width='20' height='20'><path stroke='" + auditedStreetColor + "' stroke-width='3' d='M 2 10 L 18 10 z'></svg>";
+    if (params.isUserDash) {
```

Comment:
I have to also use that parameter to check if I need to call setRegionFocus after rendering the submitted labels.  I was thinking since the type of map will mean different types of things will be done on the map, it would be good to have the parameter `isUserDash`.

Reply:
I think I would argue that if there are two things that are different between the user dashboard and some other map, then those should just be split into two separate parameters. Plus, you've probably seen that I suggested that we can just always include the `setRegionFocus()` bit.

Reply:
Even if the map is not the admin/user dashboard view setRegionFocus() can still be run?